Flapjack/CoreData/CoreDataSource.swift

```diff
@@ -313,9 +319,17 @@ public class CoreDataSource<T: NSManagedObject & DataObject>: NSObject, NSFetche
     private func contextWillBeDestroyed(_ notification: Notification) {
         guard controller.managedObjectContext === notification.object as? NSManagedObjectContext else { return }
 
+        // Flag ourselves as existing under a zombie context (one about to be fully banished) so
+        //   that when we invoke an on-change block, any objects listening to us that _ask_ us for
+        //   our number of objects or our array of objects will get back empty ones (so as to
+        //   preserve data source functional integrity for collection/table view data sources).
+        // If an object needs to get a specific element from us based on these indices, that object
+        //   can still get it before it fully goes away using `object(at:)`.
+        isContextAZombie = true
```

Comment:
You may want to remove the same line from the end of this function that also sets `isContextAZombie` to be `true.

Reply:
I blame the DST change for me forgetting this.